doc/getting-started/understanding-config-files.md

```diff
@@ -6,21 +6,55 @@ Tells your node to which nodes in the network it should talk to. A minimal versi
 
 ```json
 {
-  "Producers": [
+  "LocalProducers": {
+    "groups": [
+      {
+        "localRoots": {
+          "addrs": [
+            {
+              "addr": "x.x.x.x",
+              "port": 3001
+            }
+          ],
+          "advertise": false
+        },
+        "valency": 1
+      }
+    ]
+  },
+  "PublicProducers": [
     {
-      "addr": "x.x.x.x",
-      "port": 3001,
-      "valency": 1
+      "publicRoots" : {
+        "addrs": [
+          {
+            "addr": "y.y.y.y",
+            "port": 3002
+          }
+        ],
+        "advertise": false
+      }
     }
-  ]
+  ],
+  "useLedgerAfterSlot": false
```

Comment:
useLedgerAfterSlot isn't a bool , it is a slot number. Use a number less than 0 to disable.

Reply:
oops my bad!